karabo_data/tests/conftest.py

```diff
@@ -55,6 +55,17 @@ def mock_fxe_run():
         make_examples.make_fxe_run(td)
         yield td
 
+@pytest.fixture(scope='module')
+def mock_spb_run():
+    with TemporaryDirectory() as td:
+        make_examples.make_spb_run(td)
+        yield td
+
+@pytest.fixture(scope='module')
+def mock_spb_run():
```

Comment:
You have 2 fixtures with the same name

Reply:
Ups, that was not intentional, I'll delete one. Since mock runs can be raw and proc I figured that spb could be a test for prc and fxe for raw?